c_binding/lsm_plugin_ipc.cpp

```diff
@@ -2933,3 +2938,72 @@ void lsm_plug_target_port_search_filter(const char *search_key,
         }
     }
 }
+
+static void get_batteries(int rc, lsm_battery **bs, uint32_t count,
+                          Value &response)
+{
+    uint32_t i = 0;
```

Comment:
I intend to declare and initialize variables at the beginning of each function.

Using in-loop declaration requires `-std=c99`, or you will get error on RHEL 6 which use C90 by default:
```
error: ‘for’ loop initial declarations are only allowed in C99 mode
```